ecs-agent/daemonimages/csidriver/driver/node_test.go

```diff
@@ -0,0 +1,44 @@
+// Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
```

Comment:
Q: should we also add the `go build unit` tags here?

```
//go:build unit
// +build unit
```

Reply:
yes

Reply:
Thanks for the catch!